sdk/datafactory/Azure.ResourceManager.DataFactory/tests/Scenario/DataFactoryDatasetResourceTests.cs

```diff
@@ -538,7 +541,7 @@ public async Task Dataset_HttpFile_Create()
                 {
                     RelativeUri = "fakeuri",
                     RequestMethod = "get",
-                    Format = new DatasetStorageFormat() { DatasetStorageFormatType = "Textformat" }
+                    Format = new DatasetTextFormat()
```

Comment:
the discriminator value of this is `TextFormat` which has a casing difference - this needs to re-record